src/Microsoft.ML.Transforms/Text/TokenizingByCharacters.cs

```diff
@@ -547,8 +547,30 @@ private ValueGetter<VBuffer<ushort>> MakeGetterVec(DataViewRow input, int iinfo)
     }
 
     /// <summary>
-    /// Character tokenizer splits text into sequences of characters using a sliding window.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="TokenizingByCharactersTransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Scalar of [Text](xref:Microsoft.ML.Data.TextDataViewType)  |
```

Comment:
>Scalar [](start = 35, length = 6)

I don't see any checks for that. We just check that type is text, which can be true for Scalar or Vector.